tests/Unit/NumericalAlgorithms/Spectral/CMakeLists.txt

```diff
@@ -2,7 +2,9 @@
 # See LICENSE.txt for details.
 
 set(SPECTRE_UNIT_SPECTRAL
+  NumericalAlgorithms/Spectral/Test_Basis.cpp
   NumericalAlgorithms/Spectral/Test_LegendreGaussLobatto.cpp
+  NumericalAlgorithms/Spectral/Test_LegendreGauss.cpp
```

Comment:
I suppose in alphabetical order LegendreGauss precedes LegendreGaussLobatto so I would switch them.